metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala

```diff
@@ -526,19 +424,68 @@ private class BloopPants(
       ),
       java = Some(C.Java(Nil)),
       sbt = None,
-      test = test,
+      test = bloopTestFrameworks,
       platform = Some(C.Platform.Jvm(C.JvmConfig(javaHome, Nil), None)),
-      resolution = resolution
+      resolution = None
     )
   }
 
+  val myExportClasspath: mutable.Map[String, List[Path]] =
```

Comment:
rename to something like `exportClasspathCache`? 
nit: `private val`

Reply:
done, made all following methods private